### PR TITLE
Api testing task 2

### PR DESCRIPTION
**Pull request by NadzeyaKiz**
API testing task2 is completed

### CODE REVIEW

File: code/TestAutomation.Epam.API.Tests/TechTests.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+{
+    [TestFixture]
+    public class TechTests
+    {
+        [Test]
+        //Task5 API(2)
+        public void VerifyTechResponse()
+        {
+            var response = new TechController(new CustomRestClient()).GetTechItems<List<TechItemSingleResponseModel>>();
+            Assert.That(response.Response.StatusCode, Is.EqualTo(HttpStatusCode.OK), "Invalid status code data!");
```
Comment (dnnyhlwdplc) [inline code/TestAutomation.Epam.API.Tests/TechTests.cs:18]: please mention resource as well!

File: code/TestAutomation.Epam.API/Controllers/TechController.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        /// Request that receive single tech object by id
+        /// </summary>
+        /// <typeparam name="T"><see cref="TechItemSingleResponseModel"/></typeparam>
+        /// <returns>response typeof <see cref="RestResponse"/> and <see cref="TechItemSingleResponseModel"/></returns>
+        public (RestResponse Response, T? TechModel) GetSingleTechItem<T>(string techId)
+        {
+            return Get<T>(string.Format(SingleTechResource, techId));
+        }
+
+        public (RestResponse Response, T? TechModel) AddTechItem<T>(TechItemRequestModel model)
```
Comment (dnnyhlwdplc) [inline code/TestAutomation.Epam.API/Controllers/TechController.cs:37]: please add summary :)

**Review by dnnyhlwdplc — APPROVED** [review]

File: code/TestAutomation.Epam.API/Controllers/TechController.cs (hunk 2 above)
Comment (NadzeyaKiz) [reply code/TestAutomation.Epam.API/Controllers/TechController.cs:37]: done

File: code/TestAutomation.Epam.API.Tests/TechTests.cs (hunk 1 above)
Comment (NadzeyaKiz) [reply code/TestAutomation.Epam.API.Tests/TechTests.cs:18]: done

**Pull request merged**